### PR TITLE
Update Orchestration Walkthrough

### PR DESCRIPTION
**Pull request by amessbee**
There are similar changes for two of the walkthroughs in orchestration section: [Cross Chain Swap](https://docs.agoric.com/guides/orchestration/getting-started/contract-walkthrough/cross-chain-swap.html) and [Cross Chain Unbond](https://docs.agoric.com/guides/orchestration/getting-started/contract-walkthrough/cross-chain-unbond.html). The major change is use of the new `withOrchestration` API.

**Refs:**
- Issue #1154 
- Issue #1143

### CODE REVIEW

File: main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-swap.md
Comment (turadg) [inline main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-swap.md:181]: I think we should keep explaining that the module exports a `start` function.

It's merely a convention/convenience that we define a more abstract `contract` and pass it to `withOrchestration`.

File: main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-swap.md
Comment (toliaqat) [inline main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-swap.md:180]:
```suggestion
The `contract` function when wrapped inside `withOrchestration` defines the entry point of the contract, with the arguments, `zcf`, `privateAge`, `zone`, and `tools` for orchestration:
```

File: main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-unbond.md
Comment (toliaqat) [inline main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-unbond.md:66]: I think we should delete these headings completely that are not explained yet, and add them when we know what to add for them.

File: main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-unbond.md
Comment (toliaqat) [inline main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-unbond.md:168]: Can we rewrite this sentence, because it seems weird for some reason with `as a wrapped...`.

**Review by toliaqat — APPROVED** [review]
I left a few comments. Please fix them before merging. Looks good to me.

File: main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-swap.md
Comment (amessbee) [reply main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-swap.md]: Fixed.

File: main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-unbond.md
Comment (amessbee) [reply main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-unbond.md]: Done.

File: main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-swap.md
Comment (amessbee) [reply main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-swap.md:181]: @turadg  Will wait for your comments before merge.

Comment (turadg) [reply main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-swap.md:181]: i see. lgtm

**Pull request merged**
